frontend/packages/console-app/src/components/terminal/redux/reducer.ts

```diff
@@ -0,0 +1,16 @@
+import { Map } from 'immutable';
+import { TerminalAction, Actions } from './action';
+
+export type State = Map<string, any>;
+
+export default (state: State, action: TerminalAction) => {
+  if (!state) {
+    return Map({
+      terminal: () => {},
```

Comment:
currently your state looks like this 
```
temrinal: {
       terminal: () => {}
}
```
change
```suggestion
      isExpanded: false,
```

Reply:
updated